Rakefile

```diff
@@ -2,6 +2,7 @@ require 'rake'
 require 'rspec/core/rake_task'
 require 'puppetlabs_spec_helper/rake_tasks'
 require 'puppet'
+require 'beaker-hostgenerator'
```

Comment:
This is not necessary in the Rake file as you're not actually importing any code from BHG.